### PR TITLE
Fix problem when aws account have more than 100 instances under autoscaling groups

### PR DESCRIPTION
**Pull request by alanprot**

### CODE REVIEW

File: src/autocluster_aws.erl [hunk 1]
```diff
@@ -104,16 +104,49 @@ flatten_autoscaling_datastructure(Value) ->
...
+      error
+  end.
+
+get_autoscaling_group_node_list(error, _) -> {error, instance_discovery};
+get_autoscaling_group_node_list(Instance, Tag) ->
+  case get_all_autoscaling_instances([]) of
+    {ok, Instances} ->
       case find_autoscaling_group(Instances, Instance) of
         {ok, Group} ->
+          autocluster_log:info("Fetching autoscaling = Group: ~p", [Group]),
```
Comment (gmr) [inline src/autocluster_aws.erl]: Mind changing these to ``autocluster_log:debug``?

**Comment by gmr** [comment]
Looks good, I know that tests are failing due to the previous merge. Ideally there would be new coverage for the added functionality but since this wasn't already previously covered, I'll nab the tests in my next run through the code. If you don't mind, just update the log levels and I think we're good to go.

Thanks!

**Comment by alanprot** [comment]
Thanks.

I just amended my commit updating the log levels and removing `AUTOSCALING_DESCRIBE_INSTANCES` URL.

**Comment by gmr** [comment]
Thanks!

**Pull request merged**